### PR TITLE
Created AcraTranslator examples

### PR DESCRIPTION
**Pull request by shadinua**
* created docker-compose examples:
  - `docker/docker-compose.translator-ssession-connector-grpc.yml`
  - `docker/docker-compose.translator-ssession-connector-grpc.yml`
* written small script `tests/generate_acrastruct.py`
* added optional argument `keys_dir` to `read_storage_public_key` method

### CODE REVIEW

File: tests/generate_acrastruct.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    help='Directory where keys placed (default: docker/.acrakeys/acra-writer)')
+parser.add_argument(
+    '--data', nargs='?', default='Plain text.',
+    help='Plain text to encode (default: "Plain text.")')
+args = parser.parse_args()
+
+encryption_key = read_storage_public_key(args.client_id, args.keys_dir)
+acrastruct = create_acrastruct(args.data, encryption_key)
+
+with open('testclientid_testmessage.acrastruct', 'wb') as f:
```
Comment (Lagovas) [inline tests/generate_acrastruct.py]: maybe use filename from cli args too (instead hard coded)? with default value `testclientid_testmessage.acrastruct`?)

Comment (shadinua) [reply tests/generate_acrastruct.py]: Of course. Thanks.

File: tests/generate_acrastruct.py [hunk 2]
```diff
@@ -19,10 +19,17 @@
...
     '--data', nargs='?', default='Plain text.',
     help='Plain text to encode (default: "Plain text.")')
+parser.add_argument(
+    '--out_file', nargs='?', default='',
+    help='Save AcraStruct to filename (default: "<client_id>.acrastruct")')
 args = parser.parse_args()
 
+as_out_file = args.out_file
+if as_out_file == '':
+    as_out_file = '{}.acrastruct'.format(args.client_id)
```
Comment (Lagovas) [inline tests/generate_acrastruct.py]: example how we can avoid extra if
```
default_client_id = "testclientid"
parser.add_argument(
    '--client_id', nargs='?', default=default_client_id,
    help='Client ID (default: testclientid)')
parser.add_argument(
    '--out_file', nargs='?', default='{}.acrastruct'.format(args.client_id),
    help='Save AcraStruct to filename (default: "<client_id>.acrastruct")')
```
but in general no matter)

Comment (shadinua) [reply tests/generate_acrastruct.py]: Many thanks! But I still prefer the existent code as more clear.

**Pull request merged**